build.xml

```diff
@@ -38,89 +57,258 @@ where:
-			<dirname property="xspec.base.dir" file="@{xspec.xml}" />
-			<local name="xspec.xml.base" />
-			<basename property="xspec.xml.base" file="@{xspec.xml}" suffix=".xml" />
-			<local name="xspec.base" />
-			<basename property="xspec.base" file="${xspec.xml.base}" suffix=".xspec" />
-			<antcall target="xspec.xspec" inheritall="false">
-				<param name="xspec.xml"
-							 location="@{xspec.xml}" />
-				<param name="xspec.base.dir"
-							 location="${xspec.base.dir}" />
-				<param name="xspec.dir"
-							 location="${xspec.base.dir}/xspec" />
-				<param name="xspec.base"
-							 value="${xspec.base}" />
-			</antcall>
-		</sequential>
-	</macrodef>
+  <!-- OXYGEN PATCH needed to create a custom Saxon, for example  -warnings:recover -strip:none -opt:10 -dtd:off -l:off -versionmsg:off -expand:on -outval:fatal -val:lax -->
+  <property name="saxon.custom.options" value=""/>
+  <!-- OXYGEN PATCH END -->
+  
+  <!-- OXYGEN PATCH needed to control the cleanup of all intermediate files -->
+  <property name="clean.output.dir" value="false"/>
+  <!-- OXYGEN PATCH END -->
+  
+  <!-- OXYGEN PATCH needed to customize the HTML result. -->
+  <property name="format.xspec.report" value="${xspec.project.dir}/src/reporter/format-xspec-report.xsl"/>
+  <!-- OXYGEN PATCH END -->
+  
+  <!-- OXYGEN PATCH needed to control the cleanup of all intermediate files -->  
+  <condition property="can.delete">
+    <and>
+      <equals arg1="${clean.output.dir}" arg2="true"/>
+    </and>
+  </condition>
+  <!-- OXYGEN PATCH END -->
+  
+  
+  <!-- OUP PATCH -->
+  
+  <!-- 
+    xmltask
+    download: http://www.oopsconsultancy.com/software/xmltask/
+    help: http://www.planetgeek.ch/2009/05/17/perform-xpath-queries-inside-ant/ 
+    
+  -->
+  <taskdef name="xmltask" classname="com.oopsconsultancy.xmltask.ant.XmlTask"/>
+  <taskdef resource="net/sf/antcontrib/antlib.xml"/>
+  
+  <property name="test.type" value="t"/>
+  <condition property="test.xslt">
+    <equals arg1="${test.type}" arg2="t"/>
+  </condition>
+  <condition property="test.schematron">
+    <equals arg1="${test.type}" arg2="s"/>
+  </condition>
+  
+  <!-- Copy of original XSpec file
+  will be overwritten if XSpec applies to Schematron -->
+  <property name="xspec.compiled.xml" location="${xspec.base.dir}/${xspec.base}-compiled-oxygen.xspec"/>
+  
+  <!-- Location of compiled Schematron XSL file 
+    Can be overridden, e.g. if original Schematron references helper files at relative paths, 
+    the compiled XSL file will need to go in the same folder as the original Schematron
+   -->
+  <property name="xspec.compiled.xsl.dir" value="${xspec.dir}"/>
+  <property name="xspec.compiled.xsl" value="${xspec.compiled.xsl.dir}/${xspec.base}-sch-compiled.xsl"/>
+  <!-- for passing as a parameter to the script that generates the XSpec with @stylesheet 
+  from the XSpec with @schematron -->
+  <property name="xspec.compiled.xsl.param" value="file:/${xspec.compiled.xsl}"/>
```

Comment:
The slash in `file:/${xspec.compiled.xsl}` is problematic in Unix-like system (see full explanation in review). Try to change it with one of these two options and see which way (if any) works in Windows:
```
<property name="xspec.compiled.xsl.param" value="file:${xspec.compiled.xsl}"/>
<property name="xspec.compiled.xsl.param" value="file://${xspec.compiled.xsl}"/>
```